src/ts/ui/SignUI/UserInfoEditUI.ts

```diff
@@ -0,0 +1,81 @@
+import { basePath } from '../App';
+import { requestUpdate } from '../../domain/UserDomain/request';
+import { validateUserInfo } from '../../domain/UserDomain/validator';
+import { MESSAGE } from '../../constants/message';
+import { showSnackbar } from '../../utils';
+import { $, $$ } from '../../utils/dom';
+import { viewPainter } from '../ViewPainter';
+
+export default class UserInfoEditUI {
+  private readonly userDomain;
+
+  constructor(userDomain) {
+    this.userDomain = userDomain;
+    $('.user-info-edit__form').addEventListener('submit', this.submitHandler);
+  }
+
+  render() {
+    $('#main').classList.add('hide');
+    $('#sign-in').classList.add('hide');
+    $('#sign-up').classList.add('hide');
+    $('#user-info-edit').classList.remove('hide');
+
+    ($('#user-info-edit-email') as HTMLInputElement).value =
+      this.userDomain.userInfo.email;
+    $('#user-info-edit-name').focus();
+  }
+
+  private submitHandler = (e: SubmitEvent) => {
+    e.preventDefault();
+
+    if (!(e.target instanceof HTMLFormElement)) return;
+
+    const email = (e.target.elements.namedItem('email') as HTMLInputElement)
+      .value;
+    const name = (e.target.elements.namedItem('name') as HTMLInputElement)
+      .value;
+    const password = (
+      e.target.elements.namedItem('password') as HTMLInputElement
+    ).value;
+    const confirmPassword = (
+      e.target.elements.namedItem('confirm-password') as HTMLInputElement
+    ).value;
+
+    const user = { email, name, password };
+
+    try {
+      validateUserInfo(user, confirmPassword);
+    } catch ({ message }) {
+      showSnackbar(message);
+      return;
+    }
+
+    this.editUserInfo(user);
+  };
+
+  private editUserInfo(user: {
+    email: string;
+    name: string;
+    password: string;
+  }) {
```

Comment:
```suggestion
  private editUserInfo(user: UserInfo) {
```
인터페이스를 사용해주셨으니 인자값을 받을 때 활용하는게 좋아보이네요~

Reply:
헉 네네! any 타입 같은 몇몇 변수 타입 지정 바꾸면서 같이 변경해주었습니다 😊 알려주셔서 감사합니다~!
https://github.com/woowacourse/javascript-vendingmachine/pull/43/commits/6f481829d45ada0d180ee46e6a12f794eb43e1e4